### PR TITLE
Observability build

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- [x] Built Oberservability page

- [x] Built Observability form

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/observability or in the demo link: https://ubuntu-com-10105.demos.haus/observability
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare page against [copy-doc](https://docs.google.com/document/d/1NxJdT3BD6EWnpsu-eathNyranQVImnDGhuCjzsIWh4s/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4242

### CODE REVIEW

**Comment by petesfrench** [comment]
DEMO: https://ubuntu-com-10105.demos.haus/observability

**Comment by spencerbygraves** [comment]
Thanks @petesfrench, a couple of things...

- Can you make the subtitle "Observability with open source..." h2 please
- Two images are missing here:
image
- Can we give the 3 logo headers the same colour and spacing please, the second one looks about right:
image
- Can you use [this Openstack logo](https://assets.ubuntu.com/v1/5a3b2944-Openstack-logo-2021.svg) in the last logo section please
- It looks like the bullet points in the last section are [the indented style](https://vanillaframework.io/docs/examples/base/lists/unordered-list), can they be the solid ones please

Thanks.

**Comment by petesfrench** [comment]
> * It looks like the bullet points in the last section are [the indented style](https://vanillaframework.io/docs/examples/base/lists/unordered-list), can they be the solid ones please

@sowasred2012 Hey Scott, is there a way this can be acheived, as the list needs to be nested within the list of the parent matrix?

**Comment by petesfrench** [comment]
@spencerbygraves How important is it that the bullet points are not in the indented style? It is possible but would be a bit hacky..

File: templates/observability/index.html [hunk 1]
```diff
@@ -0,0 +1,519 @@
...
+        <p><a href="/observability/contact-us" class="p-button--positive js-invoke-modal">Contact Us</a></p>
+      </div>
+      <div class="col-5 u-align--center u-vertically-center">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/e68fd355-1+discovery_AW.svg",
+          alt="",
+          width="403",
+          height="230",
+          hi_def=True,
+          loading="lazy"
```
Comment (sowasred2012) [inline templates/observability/index.html:28]: This image is "above the fold", so this value shoud be `auto` or `eager`, to reduce the chance of layout shift when the image loads. See `image-template` [docs](https://github.com/canonical-web-and-design/canonicalwebteam.image-template#parameters)

File: templates/observability/index.html [hunk 2]
```diff
@@ -0,0 +1,519 @@
...
+    <div class="p-logo-section u-sv3">
+      <h2 class="u-align--center p-logo-section__title">A SELECTION OF SUPPORTED OPEN SOURCE MONITORING TOOLS AND DASHBOARDS</h2>
+    </div>
+  </div>
+  <div class="row p-divider">
+    <div class="col-3 p-divider__block">
+      <h3>Metrics</h3>
+      <div class="p-heading-icon--small">
+        <div class="p-heading-icon__header">
+          <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/826d5be3-1200px-Prometheus_software_logo.svg.png" alt="">
```
Comment (sowasred2012) [inline templates/observability/index.html:56]: All of these images should use the image template

File: templates/observability/index.html [hunk 3]
```diff
@@ -0,0 +1,519 @@
...
+    <p><a href="/managed/contact-us" class="p-button--positive">Request a free deployment assessment</a></p>
+    <hr class="p-separator" />
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top u-no-padding--bottom">
+  <div class="u-fixed-width">
+    <ul class="p-matrix">
+      <li class="p-matrix__item">
+        <img class="p-matrix__img" src="https://assets.ubuntu.com/v1/3b176ef1-managed-private-cloud.svg" alt="">
```
Comment (sowasred2012) [inline templates/observability/index.html:286]: These too should use the image template

File: templates/observability/index.html [hunk 4]
```diff
@@ -0,0 +1,519 @@
...
+    </div>
+    <hr class="p-separator" />
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="u-fixed-width">
+    <ul class="p-matrix">
+      <li class="p-matrix__item">
+        <img class="p-matrix__img" src="https://assets.ubuntu.com/v1/7de21966-snapshot.svg" alt="">
```
Comment (sowasred2012) [inline templates/observability/index.html:456]: Same thing - image template, rather than img tags

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
We should be using the image template for all images, it helps keep pages performant - check out [this discourse doc](https://discourse.ubuntu.com/t/editor-tricks/14448) if you haven't already, you can add a code snippet to VS code that makes adding image template blocks really easy.

**Comment by pmahnke** [comment]
@petesfrench also, the asset server also has image tags to copy 

image

**Comment by sowasred2012** [comment]
> @petesfrench also, the asset server also has image tags to copy
> 
> image

:exploding_head: didn't know that! Very cool.

**Comment by petesfrench** [comment]
@sowasred2012 Thank you for the feedback, I have made those changes now

File: templates/observability/index.html [hunk 5]
```diff
@@ -0,0 +1,675 @@
...
+    <p>Future-proof your open source observability with Ubuntu Advantage: a single, per-node package of the most comprehensive enterprise security and support for open source infrastructure and applications, including a variety of best-in-class monitoring tools and monitoring dashboards.</p>
+    <a href="/advantage/subscribe" class="p-button--positive">Buy Ubuntu Advantage</a>
+    <a href="/support">What is Ubuntu Advantage?</a>
+  </div>
+</section>
+
+<section class="p-strip u-hide--small">
+  <div class="u-fixed-width">
+    <div class="p-logo-section u-sv3">
+      <h2 class="u-align--center p-logo-section__title">A SELECTION OF SUPPORTED OPEN SOURCE MONITORING TOOLS AND DASHBOARDS</h2>
```
Comment (sowasred2012) [inline templates/observability/index.html:48]:
```suggestion
      <h2 class="u-align--center p-logo-section__title">A selection of supported open source monitoring tools and dashboards</h2>
```

File: templates/observability/index.html [hunk 6]
```diff
@@ -0,0 +1,675 @@
...
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+      <h3>Regain focus</h3>
+      <ul>
+        <li>You drive the business, we maintain the monitoring tools</li>
+        <li>Put DevOps on-rails with high quality observability</li>
+        <li>Expand globally, without needing a global team</li>
+      </ul>
```
Comment (sowasred2012) [inline templates/observability/index.html:613]:
```suggestion
      <ul class="p-list">
        <li class="p-list__item is-ticked">You drive the business, we maintain the monitoring tools</li>
        <li class="p-list__item is-ticked">Put DevOps on-rails with high quality observability</li>
        <li class="p-list__item is-ticked">Expand globally, without needing a global team</li>
      </ul>
```

File: templates/observability/index.html [hunk 7]
```diff
@@ -0,0 +1,675 @@
...
+        loading="lazy"
+        ) | safe
+      }}
+      <h3>Reliable and robust</h3>
+      <ul>
+        <li>Reduce human-error</li>
+        <li>Automated at scale</li>
+        <li>Powered by <a class="p-link--external" href="https://juju.is/">Juju</a></li>
+        <li>Managed by app-specific experts</li>
+      </ul>
```
Comment (sowasred2012) [inline templates/observability/index.html:631]:
```suggestion
      <ul class="p-list">
        <li class="p-list__item is-ticked">Reduce human-error</li>
        <li class="p-list__item is-ticked">Automated at scale</li>
        <li class="p-list__item is-ticked">Powered by <a class="p-link--external" href="https://juju.is/">Juju</a></li>
        <li class="p-list__item is-ticked">Managed by app-specific experts</li>
      </ul>
```

File: templates/observability/index.html [hunk 8]
```diff
@@ -0,0 +1,675 @@
...
+        loading="lazy"
+        ) | safe
+      }}
+      <h3>Predictable pricing</h3>
+      <ul>
+        <li>Per-machine price</li>
+        <li>Multiple portfolio apps, no extra cost</li>
+        <li>Unlimited usage</li>
+        <li>Transfer management back on request</li>
+      </ul>
```
Comment (sowasred2012) [inline templates/observability/index.html:649]:
```suggestion
      <ul class="p-list__item">
        <li class="p-list__item is-ticked">Per-machine price</li>
        <li class="p-list__item is-ticked">Multiple portfolio apps, no extra cost</li>
        <li class="p-list__item is-ticked">Unlimited usage</li>
        <li class="p-list__item is-ticked">Transfer management back on request</li>
      </ul>
```

File: templates/observability/index.html [hunk 9]
```diff
@@ -0,0 +1,675 @@
...
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+      <h3>Open source, simplified</h3>
+      <ul>
+        <li>Bring your own cloud</li>
+        <li>The latest open source dashboards and tools, today, without hiring delays</li>
+        <li>Offload resource-intensive operations to our specialist team</li>
+      </ul>
```
Comment (sowasred2012) [inline templates/observability/index.html:666]:
```suggestion
      <ul class="p-list">
        <li class="p-list__item is-ticked">Bring your own cloud</li>
        <li class="p-list__item is-ticked">The latest open source dashboards and tools, today, without hiring delays</li>
        <li class="p-list__item is-ticked">Offload resource-intensive operations to our specialist team</li>
      </ul>
```

**Review by sowasred2012 — COMMENTED** [review]
@petesfrench almost there - one grammar thing, and a tweak to the lists, it's my fault for not mentioning @spencerbygraves had requested these lists be `is-ticked`, apologies

**Comment by spencerbygraves** [comment]
@petesfrench thanks for the updates 👍

**Review by pmahnke — CHANGES_REQUESTED** [review]
getting there... some comments below

**Comment by mmanciop** [comment]
- There is a *lot* of whitespace between sections, e.g., above `TRUSTED BY`.
- I am a bit surprised to see the `A SELECTION OF SUPPORTED OPEN SOURCE MONITORING TOOLS AND DASHBOARDS` section on a different background than the item above that it belongs to, namely "Reliable monitoring tools you can count on"
- There is a `[ Request a free deployment assessment  ]` missing below the `OPTIMISE YOUR OPEN SOURCE OBSERVABILITY` header

**Comment by mmanciop** [comment]
I am also not sold on the line breaks (`<hr class="p-separator"/>`) in between most sections

**Comment by mmanciop** [comment]
Finally, I realize the design in Google Docs misses a "final Get in touch with us" CTA, I made a few updates to the [Google Doc](https://docs.google.com/document/d/1NxJdT3BD6EWnpsu-eathNyranQVImnDGhuCjzsIWh4s/edit#). The changes are: the final "closing" section, and some minor changes to the headers inside the "Managed monitoring tools, anywhere" section.

**Comment by mmanciop** [comment]
Finally, I realize the design in Google Docs misses a "final Get in touch with us" CTA, I made a few updates to the [Google Doc](https://docs.google.com/document/d/1NxJdT3BD6EWnpsu-eathNyranQVImnDGhuCjzsIWh4s/edit#). The changes are: the final "closing" section, and some minor changes to the headers inside the "Managed monitoring tools, anywhere" section.

**Comment by mmanciop** [comment]
Overall, can I please be added as reviewer? :-)

**Comment by mmanciop** [comment]
I just clicked on the contact form and... well, it does not look right folks.
Screenshot from 2021-08-04 11-55-13

**Comment by mmanciop** [comment]
The last page of the contact form also has a rather strange flow when compared to other contact forms, say, the OpenStack one: https://ubuntu.com/openstack#get-in-touch

Screenshot from 2021-08-04 11-57-14

**Comment by petesfrench** [comment]
@spencerbygraves What do you think of the design suggestions from @mmanciop ?

**Comment by spencerbygraves** [comment]
@mmanciop

> There is a lot of whitespace between sections, e.g., above TRUSTED BY.

We use strips that come with different amounts of padding, we'll have another look at which ones to use.

> I am a bit surprised to see the A SELECTION OF SUPPORTED OPEN SOURCE MONITORING TOOLS AND DASHBOARDS section on a different background than the item above that it belongs to, namely "Reliable monitoring tools you can count on"

That wasn't completely clear from the copy doc but I will update the design.

> I am also not sold on the line breaks...

We use these to either clearly separate content, or give structure to related content.

> In the contact form and the padding around the listy items does look much larger than some other contact forms with comparable structure.

I agree it could be improved, we'll update it to be more like this one - https://ubuntu.com/openstack/managed#get-in-touch

**Comment by mmanciop** [comment]
> @mmanciop
> 
> > There is a lot of whitespace between sections, e.g., above TRUSTED BY.
> 
> We use strips that come with different amounts of padding, we'll have another look at which ones to use.
 
Thanks!

> > I am a bit surprised to see the A SELECTION OF SUPPORTED OPEN SOURCE MONITORING TOOLS AND DASHBOARDS section on a different background than the item above that it belongs to, namely "Reliable monitoring tools you can count on"
> 
> That wasn't completely clear from the copy doc but I will update the design.

I would be very interested in understanding the best practices for the Google Docs, so that I can make it clearer the next time. How would I have documented something like this?

> > I am also not sold on the line breaks...
> 
> We use these to either clearly separate content, or give structure to related content.

I understand the rationale; my remark, admittedly poorly worded, was more in the direction of finding the amount of separation too large for things that were related with one another. I can leave some comments in the HTML for which separators I found, well, too successful at separating, I guess :D

> > In the contact form and the padding around the listy items does look much larger than some other contact forms with comparable structure.
> 
> I agree it could be improved, we'll update it to be more like this one - https://ubuntu.com/openstack/managed#get-in-touch

File: templates/observability/index.html [hunk 10]
```diff
@@ -0,0 +1,647 @@
...
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--bottom">
+  <div class="u-fixed-width">
+    <h2>Managed monitoring tools, anywhere</h2>
+    <p class="p-heading--four">You focus on your apps, we run the tools for you</p>
+    <p>Great monitoring tools enable you to greatly improve your applications, but are hard and time-consuming to run reliably and at scale. Let us run the best open source monitoring tools for you, so you can leverage them to continuously improve your own applications.</p>
+    <p><a href="/managed/contact-us" class="p-button--positive">Request a free deployment assessment</a></p>
+    <hr class="p-separator" />
```
Comment (mmanciop) [inline templates/observability/index.html:368]: This separator feels a bit too large, as we are qualifying with the grid below why it is a good idea for the prospect to get their monitoring tools to be run by Canonical

File: templates/observability/index.html [hunk 11]
```diff
@@ -0,0 +1,647 @@
...
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--bottom u-no-padding--top">
+  <div class="u-fixed-width">
+    <h2 class="p-heading--four">We run your monitoring tools anywhere</h2>
+    <p>Your monitoring tools and monitoring dashboards, run by Canonical, wherever your applications are.</p>
+    <p><a href="/managed/contact-us" class="p-button--positive">Request a free deployment assessment</a></p>
+    <hr class="p-separator" />
```
Comment (mmanciop) [inline templates/observability/index.html:465]: This separator feels too large, the "We run your monitoring tools anywhere" is substantiated by the logos of all the places we do that, and now it feels too far away in the page

File: templates/observability/index.html [hunk 12]
```diff
@@ -0,0 +1,661 @@
...
+    </div>
+  </div>
+</section>
+
+<div class="u-fixed-width">
+  <hr />
+</div>
+
+<section class="p-strip">
+  <div class="row">
```
Comment (mmanciop) [inline templates/observability/index.html:554]: @petesfrench, please add here:

```
<p class="p-logo-section__title u-align--center">Open source monitoring you can lean on</p>
```

As per https://docs.google.com/document/d/1NxJdT3BD6EWnpsu-eathNyranQVImnDGhuCjzsIWh4s/edit#heading=h.d8od1qesrheo

**Review by sowasred2012 — APPROVED** [review]
LGTM

**Pull request merged**